src/main/java/xyz/vegaone/easytracking/dao/BugDaoImpl.java

```diff
@@ -0,0 +1,37 @@
+package xyz.vegaone.easytracking.dao;
+
+import org.springframework.stereotype.Repository;
+import xyz.vegaone.easytracking.domain.Bug;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.transaction.Transactional;
+
+@Repository
+public class BugDaoImpl implements BugDao {
+
+    @PersistenceContext
+    private EntityManager entityManager;
+
+    @Override
+    public Bug getBug(Long id) {
+        return entityManager.find(Bug.class, id);
+    }
+
+    @Override
+    @Transactional
```

Comment:
I think it's save to move `@Transactional` at the class level. This way each method will be covered in a transaction.

Reply:
pot face modificarea de aici ? Sau din IDE ?